pkg/playbook/app/domain/installation.go

```diff
@@ -51,9 +51,10 @@ type Installation struct {
 	KeyFile             string `yaml:"keyFile,omitempty"`
 	KeyPassword         string `yaml:"keyPassword,omitempty"`
 	// Deprecated: Location is deprecated in favor of CAPILocation. It will be removed on a future release
-	Location    string             `yaml:"location,omitempty"`
-	P12Password string             `yaml:"p12Password,omitempty"`
-	Type        InstallationFormat `yaml:"format,omitempty"`
+	Location     string             `yaml:"location,omitempty"`
+	P12Password  string             `yaml:"p12Password,omitempty"`
+	UseLegacyP12 bool               `yaml:"useLegacyPkcs12,omitempty"`
```

Comment:
can we change the yaml label to `useLegacyP12` to be consistent with the other P12 flag `p12Password`.

Reply:
this change is done by latest commit